django/Dockerfile

```diff
@@ -11,8 +11,9 @@ ARG IMAGETYPE
 RUN groupadd --system django && useradd --system --gid django django
 
 # Install dependencies
-RUN apt-get update \
-    && apt-get install -y gcc \
+ RUN apt-get update && apt-get install -y gcc build-essential && apt-get install -y python3-cffi libcairo2 python3-cffi libcairo2 libpango-1.0-0 \
+    libpangocairo-1.0-0 libgdk-pixbuf2.0-0 \
+    libffi-dev shared-mime-info \
```

Comment:
Why the weird line breaks? Put each command on its own line and then break lines only when the line crosses 80 characters.

Reply:
```Dockerfile
RUN apt-get update
    && apt-get install -y <all your packages> \
        <more packages> \
        <even more packages>
```

For best results sort packages alphabetically.